Topic 0/src/point2/Builder.java

```diff
@@ -0,0 +1,14 @@
+package point2;
+
+
+public interface Builder {
+	
+	public abstract void buildRoom();
```

Comment:
[High] Don't declare abstract methods for interfaces. All classes that implement this interface should specify the behaviour for each method.